src/Microsoft.DocAsCode.Dfm/MarkdownValidators/MarkdownSytleDefinition.cs

```diff
@@ -0,0 +1,20 @@
+﻿// Copyright (c) Microsoft. All rights reserved.
+// Licensed under the MIT license. See LICENSE file in the project root for full license information.
+
+namespace Microsoft.DocAsCode.Dfm.MarkdownValidators
+{
+    using System.Collections.Generic;
+
+    using Newtonsoft.Json;
+
+    public class MarkdownSytleDefinition
```

Comment:
tagRules is an object, each one has an id, and they will be treat as a part of rule with the name of category, or a rule with name of full id (category:id)

---
In reply to: [72753608](https://github.com/dotnet/docfx/pull/545#discussion_r72753608) [](ancestors = 72753608)